libs/contact/driver/zendesk/src/config.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, Inject } from '@angular/core';
+import {
+	DaffContactZendeskConfigToken,
+	DaffContactZendeskConfig,
+} from './config';
+
+@Injectable()
+export class DaffContactZendeskConfigService {
```

Comment:
This service seems unnecessary. Since this transformation is only needed in the driver service, it could just be a function in the driver service instead of its own service.

Second question: Why not have the the `DaffContactZendeskConfigToken` have the whole endpoint rather than just the subdomain? Is there a reason for it to be just the subdomain?

Reply:
That's a really good point, I think the idea here would be to encapsulate changes to the Zendesk domain, but I guess that if it changes, we'd likely need to write a new driver, so maybe it is pointless.

Reply:
@lderrickable this is resolved in the latest commit.